README.md

```diff
@@ -28,14 +26,22 @@ Running `nina` requires:
 
 Install instruction:
 
+Find url to latest `whl`
+[here](https://github.com/MindTooth/fish-code/releases/latest) for use in
+command below.
+
 ```sh
 # install option 1
 virtualenv .venv --download  # or `python -m venv .venv --upgrade-deps`
 . .venv/bin/activate
-pip install <url-to-repo-or-whl>
+pip install '<url>'[cpu]
```

Comment:
Having issue installing via `pip` when adding [cpu] at the back.

```
pip install 'https://github.com/MindTooth/fish-code/releases/download/v1.0.0-rc2/nina-1.0.0rc2-py3-none-any.whl'[cpu]
# or
pip install https://github.com/MindTooth/fish-code/releases/download/v1.0.0-rc2/nina-1.0.0rc2-py3-none-any.whl[cpu]
```

After some testing now on Windows, we need to alter it to first make sure `wheel` is installed.  Does not seem to be bundled with the installer from python.org and the command needs to be:
```
pip install 'nina[cpu] @ https://github.com/MindTooth/fish-code/releases/download/v1.0.0-rc2/nina-1.0.0rc2-py3-none-any.whl'
```

I'll go ahead at updating the docs to reflect this.